src/framework/mpas_constants.F

```diff
@@ -22,18 +22,18 @@ module mpas_constants
 
    use mpas_kind_types
 
-   real (kind=RKIND), parameter :: pii     = 3.141592653589793   !< Constant: Pi
-   real (kind=RKIND), parameter :: a       = 6371229.0           !< Constant: Spherical Earth radius [m]
-   real (kind=RKIND), parameter :: omega   = 7.29212e-5          !< Constant: Angular rotation rate of the Earth [s-1]
-   real (kind=RKIND), parameter :: gravity = 9.80616             !< Constant: Acceleration due to gravity [m s-2]
-   real (kind=RKIND), parameter :: rgas    = 287.0               !< Constant: Gas constant for dry air [J kg-1 K-1]
-   real (kind=RKIND), parameter :: rv      = 461.6               !< Constant: Gas constant for water vapor [J kg-1 K-1]
-   real (kind=RKIND), parameter :: rvord   = rv/rgas             !
-!  real (kind=RKIND), parameter :: cp      = 1003.0              !< Constant: Specific heat of dry air at constant pressure [J kg-1 K-1]
-   real (kind=RKIND), parameter :: cp      = 7.*rgas/2.          !< Constant: Specific heat of dry air at constant pressure [J kg-1 K-1]
-   real (kind=RKIND), parameter :: cv      = cp - rgas           !< Constant: Specific heat of dry air at constant volume [J kg-1 K-1]
-   real (kind=RKIND), parameter :: cvpm    = -cv/cp              !
-   real (kind=RKIND), parameter :: prandtl = 1.0                 !< Constant: Prandtl number
+   real (kind=RKIND), parameter :: pii     = 3.141592653589793_RKIND  !< Constant: Pi
+   real (kind=RKIND), parameter :: a       = 6371229.0_RKIND          !< Constant: Spherical Earth radius [m]
+   real (kind=RKIND), parameter :: omega   = 7.29212e-5_RKIND         !< Constant: Angular rotation rate of the Earth [s-1]
+   real (kind=RKIND), parameter :: gravity = 9.80616_RKIND            !< Constant: Acceleration due to gravity [m s-2]
+   real (kind=RKIND), parameter :: rgas    = 287.0_RKIND              !< Constant: Gas constant for dry air [J kg-1 K-1]
+   real (kind=RKIND), parameter :: rv      = 461.6_RKIND              !< Constant: Gas constant for water vapor [J kg-1 K-1]
+   real (kind=RKIND), parameter :: rvord   = rv / rgas                !
+!  real (kind=RKIND), parameter :: cp      = 1003.0_RKIND             !< Constant: Specific heat of dry air at constant pressure [J kg-1 K-1]
```

Comment:
Do we want to use this PR as an opportunity to remove the commented out `cp`? Or should we move that for another PR? Or do we want to keep the commented out `cp`?

Reply:
I don't know why the commented-out value for `cp` was left in the code, so I think I'd prefer to just leave it in place, and to have this PR focus on converting all literal constants to the correct kind-type.

Reply:
@mgduda Sounds good!